examples/pom.xml

```diff
@@ -67,39 +61,33 @@
 
     <developers>
         <developer>
-            <email>santiago.pericasgeertsen@oracle.com</email>
-            <id>Santiago</id>
-            <name>Santiago Pericas-Geertsen</name>
-            <organization>Oracle</organization>
-            <roles>
-                <role>JAX-RS Spec Lead</role>
-            </roles>
-            <timezone>EST</timezone>
+            <id>developers</id>
+            <name>JAX-RS API Developers</name>
+            <email>jaxrs-dev@eclipse.org</email>
+            <url>https://github.com/eclipse-ee4j/jaxrs-api/graphs/contributors</url>
```

Comment:
Why are you doing this? This has nothing to do with "preparing examples".

Reply:
I have aligned it with the same text found at `jaxrs-api/pom.xml` and wonder what your actual problem is, as I do see this alignment as a step towards "preparing the examples folder". Why do you want it differently for examples than for jaxrs-api POM?